src/engine.cpp

```diff
@@ -53,42 +54,57 @@ const BlockSpecification &Engine::GetPhyloModelBlockSpecification() const {
 
 std::vector<double> Engine::LogLikelihoods(
     const UnrootedTreeCollection &tree_collection,
-    const EigenMatrixXdRef phylo_model_params, const bool rescaling) const {
+    const EigenMatrixXdRef phylo_model_params, const bool rescaling,
+    const std::optional<PhyloFlags> flags /*= std::nullopt*/) const {
```

Comment:
Did you just want to indicate that nullopt is the default in the cpp file?

Reply:
Yes, that was the intent.  I can remove it if you think it is overly verbose.

Reply:
I'd prefer to drop it if it's the same to you. Knowing about the default seems like something that the caller should know, and they should be reading the `.hpp` file.

Reply:
Will do.